src/core/net/ip6_address.cpp

```diff
@@ -69,6 +69,11 @@ void Prefix::Set(const uint8_t *aPrefix, uint8_t aLength)
     mLength = aLength;
 }
 
+void Prefix::SetSubnetId(uint16_t aSubnetId)
+{
+    mPrefix.mFields.m16[3] = HostSwap16(aSubnetId);
```

Comment:
Can we inline this (move to header) since it is simple one line statement?

Reply:
Updated.